### PR TITLE
format(all): JSON formatting

### PR DESCRIPTION
**Pull request by kehiy**
The JSON format is used commonly in different NIPs and the correction of JSONs and the correct Markdown format is important when someone reads them or uses them for implementation, testing, and more.

In this PR I've just fixed some broken JSON formats and used `jsonc` for JSON examples with comments to make all of them integrated with NIP-01!

### CODE REVIEW

File: 96.md [hunk 1]
```diff
@@ -143,9 +143,9 @@ The upload response is a json object as follows:
   // and, optionally, all file metadata the server wants to make available
   //
   // nip94_event field is absent if unsuccessful upload
-  nip94_event: {
+  "nip94_event2":{
```
Comment (fiatjaf) [inline 96.md:146]: Is this correct?

**Comment by fiatjaf** [comment]
This is pretty good. Thank you very much.

File: 96.md (hunk 1 above)
Comment (kehiy) [reply 96.md:146]: Based on online formatters it seems yes. But it won't affect how encoders interpret it. so I think we can go with the same way like all other JSONs in this repo.

Comment (kehiy) [reply 96.md]: Fixed.

**Comment by kehiy** [comment]
@fiatjaf You'r welcome. Changes were applied and one missed fixed corrected.

**Comment by kehiy** [comment]
Some notes on this PR and #1471  :

These kind of changes make NIPs more clean and easy to read. but we have to consider we should keep in mind to review this stuff to avoid huge refactors like this in the future. A small change request on PRs with an invalid header or JSON format for example.

**Review by staab — APPROVED** [review]

**Pull request merged**